missions/HVR_Missions.txt

```diff
@@ -1954,7 +2609,7 @@ hvr_slot4_branch = {
 					religion = ROOT
 					owned_by = ROOT
 				}
-				amount = 35
+				amount = 300 # starts at 124
```

Comment:
Probably slightly less? Maybe 250, so is just doubling starting count.

Might be too easy though if Chireiden or... Sunflower field swap, ig. Or a bit hard if Makai leaves. Not sure.

Reply:
reduced to 250 and removed owned by condition, should be easily achievable with a full conquest of Makai by senkai and heaven